reproducibility_project/src/engines/mcccs/conv_traj.py

```diff
@@ -0,0 +1,113 @@
+"""Reads the .xyz files for different seeds in NPT MC simulations, and combines and saves them in the hdf5 format."""
+import os
+import shutil
+from glob import glob
+
+import freud
+import matplotlib.pyplot as plt
+import mdtraj as md
+import numpy as np
+import signac
+from scipy import stats
+
+
+def main():
+    """Read xyz files from each replica and combine them all into one big trajectory."""
+    data_path = "analysis_data"
+    # if os.path.exists(data_path):
+    #    shutil.rmtree(data_path)
+    # os.makedirs(data_path)
+    # delete the folder manually
+    if not os.path.isdir(data_path):
+
+        os.makedirs(data_path)
+
+    os.chdir(data_path)
+
+    project = signac.get_project()
+
+    for (molecule, ensemble, temperature, pressure), group in project.groupby(
+        (
+            "molecule",
+            "ensemble",
+            "temperature",
+            "pressure",
+        )
+    ):
+        print(molecule, ensemble, temperature, pressure)
+        if not os.path.isdir(
+            "{}_{}_{}K_{}kPa".format(molecule, ensemble, temperature, pressure)
+        ):
+
+            os.makedirs(
+                "{}_{}_{}K_{}kPa".format(
+                    molecule, ensemble, temperature, pressure
+                )
+            )
+        os.chdir(
+            "{}_{}_{}K_{}kPa".format(molecule, ensemble, temperature, pressure)
+        )
+        base_dir = os.getcwd()
+        if ensemble == "NPT" and molecule == "methaneUA":
+            if os.path.exists(os.path.join(os.getcwd(), "comb_traj.h5")):
+                continue
+            traj_list = []
+            for job in group:
+                # print(job)
+                os.chdir(job.ws)
+                traj_files = sorted(glob("box1movie1a*prod*pdb*"))
```

Comment:
It looks like this glob gets pdb files.  Does it actually get xyz files?

Reply:
Thanks. Changed to .xyz. I was hoping that mdtraj will read the box lengths from the pdb file, but it didn't.